ethereum/rlp/src/main/java/tech/pegasys/pantheon/ethereum/rlp/AbstractRLPOutput.java

```diff
@@ -141,6 +141,15 @@ public int encodedSize() {
     return payloadSizes[0];
   }
 
+  /**
+   * Computes a temporary encoded data size
+   *
+   * @return The size of the RLP-encoded data written to this output.
+   */
+  public int uncheckedEncodedSize() {
```

Comment:
Can you cut this method since we don't need it and it's a bit confusing in its behavior?

Reply:
Ok.